src/test/kotlin/study/PersonTest.kt

```diff
@@ -0,0 +1,34 @@
+package study
+
+import org.assertj.core.api.AssertionsForClassTypes.assertThat
+import org.junit.jupiter.api.Test
+
+class PersonTest {
+
+    @Test
+    internal fun `이름 붙인 인자`() {
+        val person = Person(name = "황재우", age = 39, nickname = "dreamboy1339")
+        assertThat(person.name).isEqualTo("황재우")
+        assertThat(person.age).isEqualTo(39)
+        assertThat(person.nickname).isEqualTo("dreamboy1339")
+    }
+
+    @Test
+    internal fun `널 타입`() {
+        val person = Person("황재우", 39, null)
+        assertThat(person.nickname).isNull()
+    }
+
+    @Test
+    internal fun `기본 인자`() {
+        val person = Person("황재우", 39)
+        assertThat(person.name).isEqualTo(person.nickname)
+    }
```

Comment:
```suggestion
    @Test
    internal fun `기본 인자`() {
        val person = Person("황재우", 39)
        assertThat(person.name).isEqualTo("황재우")
    }
```

`person.nickname` 도 좋지만, 저는 상수로 작성된 코드가 더 잘 읽히더라구요!
취향의 영역이므로 넘어가주셔도 됩니다. 😃